packages/cmrjs/src/cmr-utils.js

```diff
@@ -66,6 +66,11 @@ function getFileDescription(file, urlType = 'distribution') {
   return filename ? `Download ${filename}` : 'File to download';
 }
 
+function isFileExtensionMatched(fileobject, extension) {
+  const filename = fileobject.key || fileobject.name || fileobject.filename || '';
```

Comment:
Just to be clear -is this distinction included because of pre-sync granules granule.file schemas?      Do we want/need this to work for those situations?